card/mod/05_standard/set/type/session.rb

```diff
@@ -0,0 +1,26 @@
+
+event :store_in_session, :after=>:approve, :on=>:create do
+  Env.session[key] = db_content
+  self.db_content = ''
+end
+
+event :update_in_session, :after=>:approve, :on=>:update do
+  if db_content_changed?
+    Env.session[key] = db_content
+    self.db_content = ''
+  end
+end
+
+event :delete_in_session, :after=>:approve, :on=>:delete do
+  Env.session[key] = nil
+  abort :success
```

Comment:
I'm not sure whether deleting a session card should delete only the session value or the card as well.